src/game/World.cpp

```diff
@@ -399,6 +399,33 @@ Weather* World::AddWeather(uint32 zone_id)
     return w;
 }
 
+void World::LoadModuleConfig()
+{
+	{
+		QueryResult_AutoPtr result = WorldDatabase.Query("SELECT `id`, `config`, `value` FROM module_config");
+		uint64 count = 0;
+
+		if (result)
+		{
+			do
+			{
+				Field* field = result->Fetch();
+				ModuleConfig mod;
+
+				uint32 id = field[0].GetUInt32();
+				mod.config = field[1].GetString();
+				mod.value = field[2].GetString();
+
+				_moduleConfig[mod.config] = mod;
+
+				count++;
+			} while (result->NextRow());
+		}
+
+		sLog.outString(">> Loaded %lu module config", count);
+	}
+}
+
```

Comment:
Yes, right here for example you are using TAB characters instead of 4 spaces.

Reply:
ok, i changed them https://github.com/talamortis/OregonCore/pull/123/commits/0e77e90f80394dd8ded03100eeff5bc3009004e1